backend/api/pretix/mutations.py

```diff
@@ -0,0 +1,65 @@
+import logging
+
+import strawberry
+from strawberry.types import Info
+
+import pretix
+from api.permissions import IsAuthenticated
+from api.pretix.query import get_user_tickets
+from api.pretix.types import AttendeeTicket, UpdateAttendeeTicketInput
+from conferences.models.conference import Conference
+
+logger = logging.getLogger(__name__)
+
+
+@strawberry.type
+class UpdateAttendeeTicketError:
+    message: str = ""
+
+
+@strawberry.type
+class TicketReassigned:
+    id: strawberry.ID
+    message: str = "Ticket was Successfully reassigned to {email}"
+
+
+UpdateAttendeeTicketResult = strawberry.union(
+    "UpdateAttendeeTicketResult",
+    (TicketReassigned, AttendeeTicket, UpdateAttendeeTicketError),
+)
+
+
+@strawberry.type
+class AttendeeTicketMutation:
+    @strawberry.mutation(permission_classes=[IsAuthenticated])
+    def update_attendee_ticket(
+        self, info: Info, conference: str, input: UpdateAttendeeTicketInput
+    ) -> UpdateAttendeeTicketResult:
+        conference = Conference.objects.get(code=conference)
+
+        try:
+            pretix.update_ticket(conference, input)
```

Comment:
make sure you add a permission check so people can't edit other people tickets

Reply:
I add a `IsOwnerPermission`